.github/scripts/pgo-build.sh

```diff
@@ -1,16 +1,24 @@
 #!/bin/bash
 
+pushd ../mmtk-openjdk/mmtk
+cargo clean
```

Comment:
Why do we need `cargo clean`? Is it to work around the compiler bug? If so, probably add a comment so we may delete them once we move to a newer version.